app/renderer/components/preferences/payment/ledgerTable.js

```diff
@@ -272,6 +272,30 @@ class LedgerTable extends ImmutableComponent {
     ]
   }
 
+  sortCheck (prevRows, currentRows) {
```

Comment:
Do you think you could add some tests or perhaps a comment block explaining the logic inside the inner loop? (Perhaps what values of each row are being compared to determine the sort check and why?) It is understandable why the old `JSON.stringify` comparison of each row may not have been efficient but it was more readable :)

Reply:
comment added